styleguide/source/_patterns/05-pages/news.json

```diff
@@ -233,37 +225,36 @@
     ],
     "learnMore": {
       "heading": {
-        "level": 5,
-        "class": "ama__h5",
         "text": "Learn More About:"
       },
       "links": [
         {
           "title": "Link to something",
           "href": "#",
           "text": "Lorem ipsum dolor sit amet.",
-          "target": "_self"
+          "target": "_self",
+          "class": "ama__link--black"
```

Comment:
This should probably not be added back?

Reply:
I didn't do anything with links per our discussion from our FED meeting this morning because it falls outside of the scope of this story.

Reply:
I agree that it's out of scope, but according to git you're adding that class here not just leaving it alone. Which is concerning.